datalad/support/annexrepo.py

```diff
@@ -13,30 +13,38 @@
 """
 
 from os import linesep
-from os.path import join as p_join, exists, normpath, isabs, commonprefix, relpath
+from os.path import join as opj, exists, normpath, isabs, commonprefix, relpath
 import logging
 
+from functools import wraps
+
 from ConfigParser import NoOptionError
 
-from gitrepo import GitRepo, files_decorator
+from gitrepo import GitRepo, normalize_paths
 from datalad.cmd import Runner as Runner
-from exceptions import CommandNotAvailableError, CommandError, FileNotInAnnexError, FileInGitError
+from exceptions import CommandNotAvailableError, CommandError, \
+    FileNotInAnnexError, FileInGitError
 
 lgr = logging.getLogger('datalad.annex')
 
 
-def _options_decorator(func):
+def kwargs_to_options(func):
```

Comment:
I'm not sure yet, too ;-)
That's why I'll leave it until we really start to make use of options and then looking into it again.